### PR TITLE
add `/redirect/paseto` to golang implementation

### PR DESCRIPTION
**Pull request by ckettell**
provides a way of entering Connect Flow using a PASETO `customer_token`

### CODE REVIEW

File: go/main.go [hunk 1]
```diff
@@ -16,6 +20,7 @@ import (
 const (
 	accessTokenURL   = "https://api.creditkudos-staging.com/v2/oauth/token"
```
Comment (ckettell) [inline go/main.go:21]: i really think these should point to our production env. Seems a bit confusing a client of ours being told to connect to staging

File: go/main.go [hunk 2]
```diff
@@ -101,6 +106,53 @@ func createRoutes(app *fiber.App, oauthConfig *oauth2.Config) {
 		c.Redirect(url)
 	})
 
+	// Setup redirection using PASETO implementation
```
Comment (jleekam) [inline go/main.go:109]:
```suggestion
	// Setup redirection using Customer Token implementation
```

File: go/main.go [hunk 3]
```diff
@@ -16,6 +20,7 @@ import (
 const (
 	accessTokenURL   = "https://api.creditkudos-staging.com/v2/oauth/token"
 	authorizationURL = "https://auth.creditkudos-staging.com/oauth/authorize"
```
Comment (jleekam) [inline go/main.go:22]:
```suggestion
	authorizationURL = "https://auth.creditkudos.com/oauth/authorize"
```

File: go/main.go [hunk 4]
```diff
@@ -101,6 +106,53 @@ func createRoutes(app *fiber.App, oauthConfig *oauth2.Config) {
 		c.Redirect(url)
 	})
 
+	// Setup redirection using PASETO implementation
+	app.Get("/redirect/paseto", func(c *fiber.Ctx) {
```
Comment (jleekam) [inline go/main.go:110]:
```suggestion
	app.Get("/redirect/customer-token-upgraded", func(c *fiber.Ctx) {
```

**Review by lucas-s-work — APPROVED** [review]

File: go/main.go [hunk 5]
```diff
@@ -14,8 +18,9 @@ import (
 
 /* #nosec G1010*/
 const (
-	accessTokenURL   = "https://api.creditkudos-staging.com/v2/oauth/token"
-	authorizationURL = "https://auth.creditkudos-staging.com/oauth/authorize"
+	accessTokenURL   = "https://api.creditkudos.com/v2/oauth/token"
```
Comment (lucas-s-work) [inline go/main.go:21]: These should be staging I think

**Review by lucas-s-work — APPROVED** [review]

**Review by jleekam — APPROVED** [review]

**Pull request merged**